packages/devextreme/testing/testcafe/tests/navigation/toolbar/common.ts

```diff
@@ -162,6 +164,15 @@ const supportedWidgets = ['dxAutocomplete', 'dxCheckBox', 'dxDateBox', 'dxMenu',
           text: 'opts.stylingMode: contained',
         },
       },
+      {
```

Comment:
This test includes scenario with dropDownButton component. We may choose other test that test appearance all components in toolbar, for example tests with names: 'all widgets render', 'nested widget appearance'. Don't forget to test this case in multiline toolbar mode.